purefa/tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+isolated_build = true
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py38
+envlist =
+    py{38}
+
+[testenv]
+ensure_default_envdir = true
+envdir =
+    py38: {toxworkdir}/py38
+dd_check_style = true
+usedevelop = true
+deps =
+    datadog-checks-base[deps]>=23.2.0
```

Comment:
```suggestion
    -e../datadog_checks_base[deps]
```

Reply:
Ack

Reply:
@sarah-witt this suggestions causes the Azure pipeline to fail. Can you confirm it is correct?

Reply:
@sarah-witt this change is causing the azure CI to fail. Can you confirm the changes we made are correct?

Reply:
Ah sorry that was my bad 🙇‍♀️ , these changes are needed for core integrations but not extras integrations. You can revert this and the requested change in `requirements-dev.txt`